heufybot/modules/commands/pronouns.py

```diff
@@ -0,0 +1,82 @@
+                                                                                                          server)]:
+            return {
+                "success": True,
+                "pronouns": self.pronouns[networkName(self.bot, server)][user.lower()]
+            }
+        if displayErrors:
+            error = "User's pronouns have not been specified. Would you like to add your own, {}?".format(user)
+            self.replyPRIVMSG(server, source, error)
+            return {
+                "success": False
+            }
+
+    def load(self):
+        self.help = "Commands: pronouns <user>, setpron <pronouns>, rmpron | "\
+                    "Query the user's pronouns or specify your own."
+        self.commandHelp = {
+            "pronouns": "pronouns <user> | Query user's pronouns",
+            "setpron": "setpron <pronouns> | Set your own pronouns.",
+            "rmpron": "rmpron | Remove your pronouns for a mysterious unspeakable reason"
+        }
+        if "pronouns" not in self.bot.storage:
+            self.bot.storage["pronouns"] = {}
+        self.pronstore = self.bot.storage["pronouns"]
+
+    def checkPermissions(self, server, source, user, command):
+        return True
+
+    def execute(self, server, source, command, params, data):
+        if command == "setpron":
+            if len(params) < 1:
+                self.replyPRIVMSG(server, source, "Your pronouns are ...blank?")
+                return
+            if networkName(self.bot, server) not in self.pronstore:
+                self.pronstore[networkName(self.bot, server)] = {}
+            self.pronstore[networkName(self.bot, server)][data["user"].nick.lower()] = " ".join(params)
+            self.bot.storage["pronouns"] = self.pronstore  # actually unsure why this is needed
+            self.bot.moduleHandler.runGenericAction("pronouns-updated", data["user"].nick, " ".join(params))
+            self.replyPRIVMSG(server, source, "Your pronouns have been noted.")
+        elif command == "rmpron":
+            if data["user"].nick.lower() not in self.pronstore[networkName(self.bot, server)]:
+                self.replyPRIVMSG(server, source, "I don't even know your pronouns!")
+            else:
+                del self.pronstore[networkName(self.bot, server)][data["user"].nick.lower()]
+                self.bot.storage["pronouns"] = self.pronstore
+                self.bot.moduleHandler.runGenericAction("pronouns-deleted", data["user"].nick)
+                self.replyPRIVMSG(server, source, "Your pronouns have been removed.")
+        elif command == "pronouns":
+            if len(params) < 1:
+                self.replyPRIVMSG(server, source, "Whose pronouns do you wanna know?")
+                return
+            # usernames probably don't need more than 1 word, right?
+            userpron = self.lookUpPronouns(server, source, params[0], True)
+            if not userpron["success"]:
+                return
+            else:
```

Comment:
This `else` keyword isn't necessary as the `if` above it ends with a `return`. I like keeping indentation as minimal as possible.